torchao/quantization/quantize_/workflows/intx/intx_unpacked_to_int8_tensor.py

```diff
@@ -207,6 +213,42 @@ def dequantize(self):
         )
 
 
+def _fake_apply_int8_act_asym_per_token_quant(hp_tensor):
```

Comment:
actually maybe a closer name for this is quantize_and_dequantize

to not confuse with fake_quant, which does not quantize to the real int values in quantize_affine